script/install.sh

```diff
@@ -78,7 +78,7 @@ fetch() {
 # ------------------------------------------------------------------------------
 
 command=
```

Comment:
While we're at it, would be great to replace the other default argument mechanism `${command:-install}` with setting command here.